tasks/main.yml

```diff
@@ -27,7 +27,9 @@
 
 - name: Select NTP provider
   set_fact:
-    timesync_ntp_provider: "{{ timesync_ntp_provider_os_default if timesync_ntp_provider_current == '' else timesync_ntp_provider_current }}"
+    timesync_ntp_provider: "{{
+        timesync_ntp_provider_os_default if timesync_ntp_provider_current == ''
+        else timesync_ntp_provider_current }}"
```

Comment:
One could use the default filter with the second parameter instead of this long if expression: https://jinja.palletsprojects.com/en/2.9.x/templates/#default